adapters/socketio_berkeley.c

```diff
@@ -954,6 +954,7 @@ void socketio_dowork(CONCRETE_IO_HANDLE socket_io)
     if (socket_io != NULL)
     {
         SOCKET_IO_INSTANCE* socket_io_instance = (SOCKET_IO_INSTANCE*)socket_io;
+        signal(SIGPIPE, SIG_IGN);
```

Comment:
When this was added, we tested it manually to make sure it was caught. I would suggest doing the same. Permission to write a smiley face. :)

Reply:
Done the manual tests you've recommended. This seems to be missing an e2e test - we could create a mock TCP server. Please open a tracking GH issue if required.